components/file_preview_modal/file_preview_modal_header/file_preview_modal_header.tsx

```diff
@@ -43,25 +48,37 @@ const FilePreviewModalHeader: React.FC<Props> = ({post, totalFiles, fileIndex, .
             />
         );
     }
+    let imageControls;
+    if (fileType === FileTypes.IMAGE || fileType === FileTypes.SVG) {
+        imageControls = (
+            <FilePreviewModalImageControls
+                toolbarZoom={toolbarZoom}
+                setToolbarZoom={setToolbarZoom}
+            />
+        );
+    }
     const actions = (
         <FilePreviewModalMainActions
             {...actionProps}
             showOnlyClose={actionProps.isMobileView}
             usedInside='Header'
         />);
     return (
-        <div className='file-preview-modal-header'>
-            {actionProps.isMobileView && actions}
-            {!actionProps.isMobileView &&
-            <FilePreviewModalInfo
-                showFileName={true}
-                post={post}
-                filename={actionProps.filename}
-            />
-            }
-            {mainActions}
-            {!actionProps.isMobileView && actions}
-        </div>
+        <React.Fragment>
+            <div className='file-preview-modal-header'>
+                {actionProps.isMobileView && actions}
+                {!actionProps.isMobileView &&
+                <FilePreviewModalInfo
+                    showFileName={true}
+                    post={post}
+                    filename={actionProps.filename}
+                />
+                }
+                {mainActions}
+                {!actionProps.isMobileView && actions}
+            </div>
+            {imageControls}
```

Comment:
I guess we could but then this happens:
<img width="1680" alt="Screenshot 2022-08-16 at 09 16 56" src="https://user-images.githubusercontent.com/108882245/184820398-899b491f-9032-4863-9259-13f3603c3ff9.png">

I could fix it but is it worth it?

Reply:
it depends ... I would prefer a tidy structure when it is just a minor adjustment

Reply:
I just remembered that now but the reason I separated them like that is because they are different divs and are stacked, it made sense for me te separate them. Please tell me if that isn't the correct way to do it and I will rectify. Alternatively, I could do that:

```
        <div>
            <div className='file-preview-modal-header'>
                {actionProps.isMobileView && actions}
                {!actionProps.isMobileView &&
                <FilePreviewModalInfo
                    showFileName={true}
                    post={post}
                    filename={actionProps.filename}
                />
                }
                {mainActions}
                {!actionProps.isMobileView && actions}
            </div>
            {imageControls}
        </div>
```

Reply:
that's not really better ... I generally like to keep the HTML clutter as minimal as possible to not create overly complex and nested structures. Let's just keep the Fragment for now and when we revisit this component we can surely improve on it then. For now it's fine and I don't want to block the PR with nitpicky stuff like this!